lib/reporting/proofing_rate_report.rb

```diff
@@ -7,10 +7,20 @@ module Reporting
   class ProofingRateReport
     DATE_INTERVALS = [30, 60, 90].freeze
 
-    attr_reader :start_date
+    attr_reader :end_date
```

Comment:
I realized, that with 30,60,90 day lookbacks... this date is the *end* of the range of time we're not looking at, the starts are what are variable. So I changed it to help read better